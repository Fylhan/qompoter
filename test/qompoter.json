{
	"name": "fylhan/qompoter-test",
	"description": "Test qompoter file",
	"keywords": ["Qt", "C++", "dependency management"],
	"license": "LGPL3+",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"trialog/solilog": "v1.0",
		"trialog/gpslib": "v1.1.1",
		"trialog/tcanp": "v1.6.7-lib",
		"ixxat/eci": "v1.8-lib"
	},
	"require-dev": {
		"trialog/autotester": "v1.0"
	},
	"repositories": [
		{
			"type": "fs",
			"url": "/media/data/Project/qompoter"
		}
	]
}
