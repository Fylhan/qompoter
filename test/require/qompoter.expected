Qompoter
======== require

* git/acme/master
* cp/fylhan/v1.0
* git/acme-tag/v1.0

done
